react-game/react-game/src/Components/AppContent.js

```diff
@@ -0,0 +1,136 @@
+import React, {useState} from 'react';
+import {Cards} from './Cards';
+import Score from './Score';
+import ScoreList from './ScoreList';
+import enterFullscreen from '../fullscreen';
+import Settings from './SettingBtn';
+
+
+
+function ModalEndGame({gameover, score, pushscore}) {
+
+  const [value, setValue] = useState('');
+
+ function onchange(e){
+    setValue(e.target.value);
+  }
+
+  return (
+   <div className={gameover ? "modal-game active" : "modal-game "}>
+    <div className={gameover ? "modal-game__content active" : "modal-game __content"} >
+    <h2>Поздравляем, вы набрали {score} очков</h2> 
+    <p>Введите ваше Имя</p>
+    <form onSubmit={(event)=> event.preventDefault()}>
+    <input className='inputscoreN' type="text" placeholder='Как вас зовут?' onChange={onchange}/><button className='btn-submitscore' type="submit" onClick={() => pushscore(value)}>Ok!</button>

```

Comment:
it is good practice to use camelCase: inputscoreN -> inputScore, onchange -> onChange